frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDasboard.tsx

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import { Form, Card, Message } from 'semantic-ui-react';
 import styles from './TeamEventCreditsForm.module.css';
 
-const REQUIRED_TEC_CREDITS = 3; // number of required tec credits in a semester
 const REQUIRED_COMMUNITY_CREDITS = 1;
 
 const TeamEventCreditDashboard = (props: {
   approvedTEC: TeamEventInfo[];
   pendingTEC: TeamEventInfo[];
+  userRole: Role;
```

Comment:
This doesn't need to be a prop since we have the `useSelf` hook 
```
const userRole = useSelf().role
```

Reply:
This causes an error because `useSelf()` can be undefined.

Reply:
https://github.com/cornell-dti/idol/blob/main/frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx#L12-L13

Reply:
Ok makes sense, thanks!